keyboards/naked48/keymaps/via_led_matrix/rules.mk

```diff
@@ -0,0 +1,5 @@
+RGBLIGHT_ENABLE = no
+RGB_MATRIX_ENABLE = yes
```

Comment:
Please rename this keymap to `via_rgb_matrix` to avoid confusion with the LED Matrix feature.

Reply:
Fixed.